gsql/cache/cache_handler.py

```diff
@@ -0,0 +1,107 @@
+import json
+import os
+from json.decoder import JSONDecodeError
+
+
+CacheDict = {}
```

Comment:
Please follow a OOP approach and encapsulate the functions and the cache dictionary

Reply:
OOps approach followed